models/public/faster_rcnn_inception_v2_coco/faster_rcnn_inception_v2_coco.md

```diff
@@ -56,7 +56,7 @@ Expected color order: BGR.
 
 ### Original Model
 
-1. Classifier, name: `detection_classes`. Contains predicted bounding boxes classes in a range [1, 91]. The model was trained on Microsoft\* COCO dataset version with 90 categories of objects.
+1. Classifier, name: `detection_classes`. Contains predicted bounding boxes classes in a range [1, 91]. The model was trained on Microsoft\* COCO dataset version with [90 categories](../../../data/dataset_classes/coco_91cl_bkgr.txt) of objects.
```

Comment:
May be `91 categories`?

Reply:
Also I think we should mention `background`

Reply:
updated both